tests/test_security_path_traversal.py

```
@@ -204,3 +205,32 @@ def test_run_tailer_path_validation(tmp_path):
+    from unittest.mock import MagicMock, patch
+
+    from fastapi import HTTPException
+    from swarm.api.routes.evolution import ApplyEvolutionRequest, apply_evolution_patch_endpoint
+
+    request = ApplyEvolutionRequest(
+        patch_id="../../etc:my-patch",
+        dry_run=True,
+        create_backup=False,
+    )
+
+    async def run_test():
```

Ruff (0.15.4)

[warning] Missing return type annotation for private function `run_test`

Add return type annotation: `None`

(ANN202)